### ISSUE TITLE
$push

### ISSUE THREAD

**Issue by devdebonair** [vdemedes/mongorito#95]
Saw a discussion about this in Issue #51. Have we reached a conclusion about adding this function to the api? I would love this feature implemented.

**Comment by vdemedes** [comment]
No, still no conclusion on this one. I have to delay this feature for future releases, in order to ship 2.0 faster.

**Comment by paul42** [comment]
while we wait for this enhancement, what's the current best practice to perform this action?

**Comment by vdemedes** [comment]
Unfortunately, none at the moment.

**Comment by ulpian** [comment]
@vdemedes makes a good point on what should be embedded and what should not, generally the example on #51 is not a good one however, this does need to be solved conceptually as other mongo methods for `$inc`, `$mul`, `$rename` etc.. (essentially all update operators https://docs.mongodb.com/v2.6/reference/operator/update-field/) would be left out of an elegant mongorito solution.

**Comment by devdebonair** [comment]
Is it possible to drop down to the native drive for the meantime?

**Comment by ulpian** [comment]
@devdebonair the issue with providing temporary solutions is that they become the norm and any future updates will have to be backward compatible, that's an issue if mongorito implements an active record style approach which will be really different to the native driver.

**Comment by devdebonair** [comment]
I see your point. So what's the major issue holding this implementation back? I can try to spin up a PR.

**Comment by ulpian** [comment]
@devdebonair please do, I guess the major issue is essentially what that update operation would look like but feel free to propose what you think is best :)
